server/src/test/scala/za/co/absa/atum/server/api/controller/CheckpointControllerSpec.scala

```diff
@@ -50,13 +50,13 @@ class CheckpointControllerSpec extends ZIOSpecDefault with TestData {
           for {
             result <- CheckpointController.createCheckpoint(checkpointDTO1)
           } yield assertTrue(result == checkpointDTO1)
-        },
+        } @@ TestAspect.tag("IntegrationTest"),
```

Comment:
Why are these considered integration tests? (multiple cases)

Reply:
My bad. I have reduced the tags.